.github/workflows/refresh.yml

```
@@ -1,27 +1,22 @@
 on:
   schedule:
-      - cron: "0 * * * *"
+  - cron: "0 * * * *"
   workflow_dispatch:
-  
+
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Retrieve
-        run: |
-          while read repository; do
-            curl --silent --location "https://hub.docker.com/v2/repositories/$repository" |
-            jq .pull_count | paste <(date +%s) - >> "$(basename "$repository")"
-          done <<-END
-            dvcorg/cml-gpu-py3-cloud-runner
-            dvcorg/cml-gpu-cloud-runner
-            dvcorg/cml-py3-cloud-runner
-            dvcorg/cml-cloud-runner
-          END
-      - name: Commit
-        run: |
-          git config --local user.name "GitHub Action"
-          git config --local user.email "action@github.com"
-          git add --all && git commit --message "Update data"
-          git push
+    - uses: actions/checkout@v2
+    - name: Retrieve
+      run: |
+        for repository in gpu-py3-cloud-runner gpu-cloud-runner py3-cloud-runner cloud-runner; do
+          curl -L "https://hub.docker.com/v2/repositories/dvcorg/cml-$repository" \
+          | jq .pull_count | paste <(date +%s) - >> "cml-$repository"
+        done
```

```suggestion
      run: >-
        for repository in gpu-py3-cloud-runner gpu-cloud-runner py3-cloud-runner cloud-runner; do
          curl --location "https://hub.docker.com/v2/repositories/dvcorg/cml-$repository"
          | jq .pull_count
          | paste <(date +%s) -
          >> "cml-$repository"
        done
```

I chose the [here document](https://en.wikipedia.org/wiki/Here_document) syntax because of the 79 character line lenght limit; line ending backslashes weren't an option. Again, that's because of the OCD I don't have. 😈 

YAML lets us, mere mortals, get creative with line breaks. 😝